styles/themes/_light.scss

```diff
@@ -243,6 +247,7 @@ html.light-mode {
 		color: $text-light-theme;
 	}
 
+	.card .card-content,
```

Comment:
![Capture d’écran 2022-09-19 à 11 57 20 AM](https://user-images.githubusercontent.com/9987732/190993502-75f80cdc-4b03-4571-bbcd-8a8c336b9268.png)

What are you trying to fix here?